cellxgene_schema_cli/tests/test_schema_compliance.py

```diff
@@ -0,0 +1,1006 @@
+        # this MUST be the most accurate HsapDv term.
+        self.validator.adata.obs["disease_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'disease_ontology_term_id' is not a "
+                "valid ontology term id of 'MONDO, PATO'"
+            ],
+        )
+
+    def test_ethnicity_ontology_term_id(self):
+
+        """
+        ethnicity_ontology_term_id categorical with str categories.
+        If organism_ontolology_term_id is "NCBITaxon:9606" for Homo sapiens,
+        this MUST be either a HANCESTRO term or "unknown" if unavailable.
+        Otherwise, for all other organisms this MUST be "na".
+        """
+
+        # If organism_ontolology_term_id is "NCBITaxon:9606" for Homo sapiens,
+        # this MUST be either a HANCESTRO term or "unknown" if unavailable.
+        self.validator.adata.obs["organism_ontology_term_id"][0] = "NCBITaxon:9606"
+        self.validator.adata.obs["ethnicity_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'ethnicity_ontology_term_id' is "
+                "not a valid ontology term id of 'HANCESTRO'"
+            ],
+        )
+
+        # Otherwise, for all other organisms this MUST be "na".
+        self.validator.errors = []
+        self.validator.adata.obs["organism_ontology_term_id"][0] = "NCBITaxon:10090"
+        self.validator.adata.obs["development_stage_ontology_term_id"][
+            0
+        ] = "MmusDv:0000003"
+        self.validator.adata.obs["ethnicity_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'ethnicity_ontology_term_id' is not a "
+                "valid value of 'ethnicity_ontology_term_id'"
+            ],
+        )
+
+    def test_organism_ontology_term_id(self):
+
+        """
+        organism_ontology_term_id categorical with str categories. This MUST be a child of NCBITaxon:33208.
+        """
+
+        # If organism_ontolology_term_id is "NCBITaxon:9606" for Homo sapiens,
+        # this MUST be either a HANCESTRO term or "unknown" if unavailable.
+        self.validator.adata.obs["organism_ontology_term_id"][0] = "EFO:0000001"
+        self.validator.adata.obs["development_stage_ontology_term_id"][0] = "unknown"
+        self.validator.adata.obs["ethnicity_ontology_term_id"][0] = "na"
+        self.validator.validate_adata()
+        self.assertEqual(
+            self.validator.errors,
+            [
+                "ERROR: 'EFO:0000001' in 'organism_ontology_term_id' is not a valid "
+                "ontology term id of 'NCBITaxon'"
```

Comment:
```suggestion
                "child ontology term id of NCBITaxon:33208 for Metazoa'"
```

Is the `NCBITaxon:33208` term itself allowed or must the term strictly be a child?

Reply:
Per the schema - This MUST be a child of NCBITaxon:33208 for _Metazoa_.

Reply:
Fixed in f68682ff4c918f36b5fae8f4fab1d2b98201caf4